src/app/components/tabBar/index.tsx

```diff
@@ -16,12 +16,21 @@ const RowContainer = styled.div((props) => ({
   alignItems: 'center',
   width: '100%',
   height: 64,
+  background: 'rgba(18, 21, 30, 0.70)',
+  backdropFilter: 'blur(12px)',
   justifyContent: 'space-between',
   paddingLeft: props.theme.spacing(30),
   paddingRight: props.theme.spacing(30),
   borderTop: `1px solid ${props.theme.colors.background.elevation3}`,
 }));
 
+const BottomBarContainer = styled.div({
+  position: 'absolute',
+  bottom: 0,
+  left: 0,
+  right: 0,
+});
+
```

Comment:
this breaks a lot of layouts because now the other components will fill up the space vacated by changing this to absolute positioning. what was the reasoning here? I would strongly suggest not doing this. look at position: fixed and position: sticky as alternatives

for example, here the scrollbar appears behind the bottom tab, and the bottom tab is covering the 'Manage Token List' button: 
<img width="356" alt="image" src="https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/67181a06-5f51-4a7d-aefe-3e048d4cb0ce">

Reply:
successfully pushed back on design for now